OpenImis.ePayment/Escape/Payment/Models/PaymentStatus.cs

```diff
@@ -0,0 +1,18 @@
+﻿
+namespace OpenImis.ePayment.Escape.Payment.Models
+{
+    public static class PaymentStatus
+    {
+        public static int FailedReconciliated  = -5;
+        public static int FailedReceived = -4;
+        public static int FailedControlNumberReceived = -3;
+        public static int FailedSent = -2;
+        public static int FailedRequested = -1;
+        public static int Cancelled = 0;
+        public static int Requested = 1;
```

Comment:
Status 1 is Payment created. Please check.

Reply:
Ok. I change this.